src/services/news.ts

```diff
@@ -101,20 +101,21 @@ const getNewsByCategory = async (category: number): Promise<NewsItem[]> =>
 
 /******************* POST **********************/
 
-async function submitStudentNews(
+const submitStudentNews = async (
   subject: string,
   categoryID: number,
   body: string,
   image?: string,
-): Promise<NewsItem | undefined> {
+): Promise<NewsItem | undefined> => {
   try {
     return await http.post(
-      `news?subject=${subject}&categoryID=${categoryID}&body=${body}&image=${image ?? ''}`,
+      `news?subject=${subject}&categoryID=${categoryID}&body=${body}`,
+      image ?? '',
```

Comment:
Hey @Kevin-Chu58, I know this is still a draft, but I wanted to note that this entire query string should be in the body, not just the image. Perhaps a `StudentNewsUpload` type would help?